src/emulator/gxm/src/gxm.cpp

```diff
@@ -512,7 +546,7 @@ void flip_vertically(uint32_t *pixels, size_t width, size_t height, size_t strid
     }
 }
 
-GLenum translate_blend_func(SceGxmBlendFunc src) {
+GLenum translate_blend_func(const SceGxmBlendFunc src) {
```

Comment:
Passed by value, doesn't need to be const.

Reply:
Oh, true.